torch/distributed/device_mesh.py

```diff
@@ -299,7 +305,7 @@ def _init_process_groups(self):
                                     f"in {subgroup_ranks}!"
                                 )
                             dim_group_infos.append(
-                                (_get_group_tag(dim_group), subgroup_ranks)
+                                (_get_group_tag(dim_group), subgroup_ranks)  # type: ignore[arg-type]
```

Comment:
nit: Curious why we have to add a `type: ignore[arg-type]` here instead of fixing the typing?

Reply:
> nit: Curious why we have to add a `type: ignore[arg-type]` here instead of fixing the typing?

Ye. Good idea. Gonna be using Chip's `not_none()` util to make sure the typing is correct.